clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -380,18 +380,21 @@ private String responseDataToLogString(FetchResponse response) {
      */
     public boolean handleResponse(FetchResponse response) {
         if (response.error() != Errors.NONE) {
-            log.info("Node {} was unable to process the fetch request with {}: {}.",
-                node, nextMetadata, response.error());
             if (response.error() == Errors.FETCH_SESSION_ID_NOT_FOUND) {
+                log.debug("Node {} was unable to process the fetch request with {}: {}. "
+                        + "This normally indicates the session has been evicted in the server side cache",
+                    node, nextMetadata, response.error());
                 nextMetadata = FetchMetadata.INITIAL;
             } else {
+                log.warn("Node {} was unable to process the fetch request with {}: {}.",
```

Comment:
Can we filter this error as well: INVALID_FETCH_SESSION_EPOCH?  This error also floods client log.

Reply:
We chatted offline. `INVALID_FETCH_SESSION_EPOCH` happens very rarely especially after #31 because we ensure the epoch is updated correctly and will not send duplicate fetch request within the same session epoch.